suggester/src/main/java/org/opengrok/suggest/util/StringUtils.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * CDDL HEADER START
+ *
+ * The contents of this file are subject to the terms of the
+ * Common Development and Distribution License (the "License").
+ * You may not use this file except in compliance with the License.
+ *
+ * See LICENSE.txt included in this distribution for the specific
+ * language governing permissions and limitations under the License.
+ *
+ * When distributing Covered Code, include this CDDL HEADER in each
+ * file and include the License file at LICENSE.txt.
+ * If applicable, add the following below this CDDL HEADER, with the
+ * fields enclosed by brackets "[]" replaced with your own identifying
+ * information: Portions Copyright [yyyy] [name of copyright owner]
+ *
+ * CDDL HEADER END
+ */
+
+/*
+ * Copyright (c) 2008, 2018, Oracle and/or its affiliates. All rights reserved.
+ * Portions Copyright (c) 2017-2018, Chris Fraire <cfraire@me.com>.
+ */
+
+package org.opengrok.suggest.util;
+
+/**
+ * Various String utility methods taken from
+ * org.opensolaris.opengrok.util.StringUtils
+ *
+ * @author austvik
+ */
+public final class StringUtils {
```

Comment:
I think we should not duplicate utility methods. I think it would be better to create some separate module which would contain that stuff.

However, there is no need for this method, you can simply use `DurationFormatUtils` (<https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/time/DurationFormatUtils.html>) which are already on the classpath.